### PR TITLE
closes #28

### LINKED ISSUE #28: yml profiles
@willingc @ellisonbg @jasongrout Based on an earlier conversation, we think it would make sense to have profiles for different talks in a yml file.  An invoke script would copy relevant files into a folder for the talk.  This way we can customize talks to different audiences.

### PR DESCRIPTION
**Pull request by jzf2101**
Starting to set up profiles in `talks.yml` 

To run call `invoke talk -t talk_name`, which will pull files and folders into a directory called `talk_name`

### CODE REVIEW

**Comment by jzf2101** [comment]
FYI my builds are failing because I need the `.travis.yml` to `install yaml`

**Comment by blink1073** [comment]
The name of the package is `pyyaml` (confusing, I know).

**Comment by blink1073** [comment]
Note that it is still imported as `yaml`, just installed as `pyyaml`.

**Comment by jzf2101** [comment]
@blink1073 build passes could you review?

File: environment.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ dependencies:
 - sympy=1.0
 - tensorflow=1.1.0
 - nodejs=6.11.0=0
+- yaml
```
Comment (blink1073) [inline environment.yml]: Should be `pyyaml`?  It looks like `pyyaml` was picked up by one of the other requirements, but it couldn't hurt to make it explicit here.

File: tasks.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
 from __future__ import print_function
 from invoke import task, run
 import os
+import yaml
+import shutil
+import pdb
```
Comment (blink1073) [inline tasks.py]: Not used anymore?

File: tasks.py [hunk 3]
```diff
@@ -70,24 +73,74 @@ def clean(ctx, env_name=env_name, demofolder=demofolder):
...
+	or in yaml format:
+
+	talk_name:
+		folders:
+			- folder0
+			- folder1
+		files:
+			- file0
+			- file1
+    rename:
```
Comment (blink1073) [inline tasks.py]: Missing indent

**Comment by willingc** [comment]
Very cool @jzf2101. I tried running locally. No problem with install of the environment. I did get the following error when trying `invoke talk -t test_talk`:

screen shot 2017-08-04 at 8 21 18 am

File: environment.yml (hunk 1 above)
Comment (jzf2101) [inline environment.yml]: Ah right

**Comment by jzf2101** [comment]
@willingc did you call `invoke demofiles` prior to `invoke talk -t test_talk`?  I suppose I could force that to be called?

**Comment by blink1073** [comment]
It seems reasonable to invoke that script first if the `demofiles` folder doesn't exist.

**Comment by willingc** [comment]
@jzf2101 No, I didn't do `invoke demofiles` first. From the yaml file, I didn't realize that was a separate step. I will double check locally with your suggested process.

**Comment by willingc** [comment]
Hi @jzf2101, Worked well when using `invoke demofiles` first 👍 

Just a heads up, I saw a couple of places where markdown titles did not have a space between the hash and the text so the headers were not rendering correctly (i.e. README.md) and notebook below:

screen shot 2017-08-04 at 1 21 16 pm

File: tasks.py (hunk 2 above)
Comment (jzf2101) [inline tasks.py]: @blink1073 Not used? I use `shutil` and `yaml` in a few lines.  I keep `pdb` in there for debugging purposes, but I could remove it I suppose?

**Comment by jzf2101** [comment]
@willingc Where in the readme did you see the missing spaces? I can't find the rendering error.

**Comment by willingc** [comment]
@jzf2101 Just checked and the errors are in the upstream repo, https://github.com/theandygross/TCGA.

**Comment by blink1073** [comment]
Looks good, thanks @jzf2101!

**Pull request merged**